soaks/bin/soak_one.sh

```diff
@@ -18,14 +19,13 @@ display_usage() {
     echo "  --variant: the variation of test in play, either 'baseline' or 'comparison'"
     echo "  --tag: the tag this test covers"
     echo "  --capture-dir: the directory in which to write captures"
-    echo "  --cpus: the total number of CPUs to dedicate to the soak minikube, default 7"
-    echo "  --memory: the total amount of memory dedicate to the soak minikube, default 8g"
     echo "  --vector-cpus: the total number of CPUs to give to soaked vector"
-    echo "  --warmup-seconds: the total number seconds to pause waiting for vector to warm up"
```

Comment:
Would it still make sense to let Vector warm up a bit? I guess we can look at the sample distribution to see if differs based on time.

Reply:
Fair point. Previously the warmup was a guess but now the place to introduce that would be in lading. Let the process run for x seconds, then start the clock on sample collection. I'll add this back in.

Reply:
Warmup now introduced in lading: https://github.com/blt/lading/pull/150